src/components/Request/FormFields/LinkedinSearchEmployers.tsx

```diff
@@ -0,0 +1,125 @@
+import { InfoLabel, Input, Text } from "@fluentui/react-components";
+import { Controller } from "react-hook-form";
+import { FormField } from "components/Request/NewRequestForm";
+import "components/Request/Request.css";
+import { TextFieldIcon } from "@fluentui/react-icons-mdl2";
+
+const rules = {
+  maxLength: {
+    value: 255,
+    message: "Employer can be no longer than 255 characters",
+  },
+};
+
+const LinkedinSearchEmployers = ({ name, form }: FormField) => {
+  return (
+    <fieldset>
+      <legend>
+        <TextFieldIcon className="requestFieldIcon" />
+        <InfoLabel weight="semibold" info="Google, Boeing, Raytheon, etc.">
+          Preferred Current/Past Employers
+        </InfoLabel>
+      </legend>
+      <Controller
+        name="linkedinSearchEmployer1"
+        control={form.control}
+        rules={rules}
+        render={({ field }) => (
+          <Input
+            {...field}
+            title="LinkedIn Search Employer 1"
+            style={{ width: "100%" }}
+            aria-describedby={name + "Err"}
+            aria-invalid={
+              form.formState.errors.linkedinSearchEmployer1 ? "true" : "false"
+            }
+          />
```

Comment:
Good point -- so you can remove it.  I suggested adding since you had 3 and 4 in there just not 1 and 2.  We do need to add in the 1-4 numbers on the Err attributes for aria-describedby and the error text itself.  For all the components like this.